repos/cli-utils/src/network/isPrivateIP.js

```diff
@@ -0,0 +1,19 @@
+const { ipIsInRange } = require('./ipIsInRange')
+
+const privateIpv4Classes = {
```

Comment:
This should go into the constants. 
* It's doubtful they will be reused, but it is possible, 
* Also, follows the pattern of other hard coded values

Reply:
Updated